Representor/HTTP/APIBlueprint/BlueprintTransition.swift

```diff
@@ -76,26 +76,33 @@ func parseAttributes(dataStructure:[String:AnyObject], builder:HTTPTransitionBui
 }
 
 extension HTTPTransition {
-  public static func from(# resource:Resource, action:Action, URL:String? = nil) -> HTTPTransition {
+  public static func from(resource  resource:Resource, action:Action, URL:String? = nil) -> HTTPTransition {
     return HTTPTransition(uri: URL ?? action.uriTemplate ?? resource.uriTemplate) { builder in
       builder.method = action.method
 
-      let addParameter = { (parameter:Parameter) -> Void in
+      func addParameter(parameter:Parameter) {
         let value = parameter.example
         let defaultValue = (parameter.defaultValue ?? nil) as NSObject?
         builder.addParameter(parameter.name, value:value, defaultValue:defaultValue, required:parameter.required)
       }
 
-      action.parameters.map(addParameter)
+      for parameter in action.parameters {
+        addParameter(parameter)
+      }
       let parameters = action.parameters.map { $0.name }
```

Comment:
the other map uses were more like a forEach, which now exists in swift 2. this case it is being used as an actual map.